src/Text/Pandoc/Readers/DocBook.hs

```diff
@@ -1058,25 +1058,44 @@ parseInline (Elem e) =
             descendantContent name = maybe "???" strContent
                                    . filterElementName (\n -> qName n == name)
 
+-- | Extract a math equation from an element
+--
+-- asciidoc can generate Latex math in CDATA sections.
+--
+-- Note that if some MathML can't be parsed it is silently ignored!
```

Comment:
"silently ignored" sounds non-ideal.
When we can't parse tex math, we:
a) issue a warning,
b) include the raw tex.
(See the LaTeX reader.)  b) might make less sense here -- nobody wants to have raw MathML in their document -- but we could still do a), right?

Reply:
I went ahead and merged  this, but it would be great if you could do another PR that adds a warning when math is being ignored, using our standard mechanism for doing that.